src/jdk.jdeps/share/classes/com/sun/tools/javap/CodeWriter.java

```diff
@@ -128,7 +129,7 @@ public void writeInstr(int pc, Instruction ins, CodeModel lr) {
             case InvokeDynamicInstruction instr ->
                 printConstantPoolRefAndValue(instr.invokedynamic(), 0);
             case InvokeInstruction instr -> {
-                if (instr.isInterface()) printConstantPoolRefAndValue(instr.method(), instr.count());
```

Comment:
Is this another change in the javap convention?

Reply:
No, that was another difference that I observed.